src/config/config.js

```diff
@@ -102,6 +102,12 @@ export const config = convict({
     }
   },
   consolidatedView: {
+    mockEnabled: {
```

Comment:
I think this should be more descriptive so its obvious what its being used eg. `mockDALEnabled` or something along those lines